README.md

```diff
@@ -251,6 +251,13 @@ checkpoints
 │   └── training_state.pth  # optimizer/scheduler/rng state and training step
 ```
 
+To resume training from a checkpoint, you can add these to the `train.py` python command:
+```bash
+    hydra.run.dir=your/original/experiment/dir resume=true
+```
+
+It will load the pretrained model, optimizer and scheduler states for training.
```

Comment:
How about adding: "For more information please see our tutorial on training resumption [here](./examples/5_resume_training.md)"

Reply:
Nice idea!